python/lsst/ctrl/ingestd/entries/entry.py

```diff
@@ -58,11 +58,14 @@ def __init__(self, butler, message, mapper):
 
         if self.file_to_ingest == dst_url:
             # Avoid E501
-            m = f"failed to map {self.file_to_ingest}; check config file for incorrect mapping"
-            raise RuntimeError(m)
+            m = f"attempt to map {self.file_to_ingest} to same file"
+            LOGGER.warning(m)
```

Comment:
A rather minor point, but I'm not quite sure why can't you just write
```python
LOGGER.warning("attempt to map %s to same file", self.file_to_ingest)
```
Even with the required indentation this line length would be way below the LSST limit (110 chars).

Reply:
Agreed.  Fixed.